packages/file_selector/file_selector_web/integration_test/file_selector_web_test.dart

```diff
@@ -1,89 +0,0 @@
-// Copyright 2020 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-// @dart = 2.9
-
-import 'dart:typed_data';
-import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
-import 'package:integration_test/integration_test.dart';
-import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
-import 'package:file_selector_web/file_selector_web.dart';
-import 'package:file_selector_web/src/dom_helper.dart';
-
-void main() {
-  group('FileSelectorWeb', () {
-    IntegrationTestWidgetsFlutterBinding.ensureInitialized();
-    MockDomHelper mockDomHelper;
-    FileSelectorWeb plugin;
-
-    setUp(() {
-      mockDomHelper = MockDomHelper();
```

Comment:
This file should definitely not be removed; a manual mock should be straightforward to write here.

Reply:
Yes I made the mock after your comment on the issue, but I'm unable to run and test it on my local machine.

Reply:
Can you elaborate on this? What's the issue with running the test?

Reply:
I opened an [issue](https://github.com/flutter/flutter/issues/76213) for this

Reply:
I've resolved this issue in the branch, the way to run the integration tests now is documented in the PR :)